NOM_visualizations/R/NOM_R_notebook.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>Can you make this plot wider?  I think if you use something like this you can adjust the sizing.  </p>

<pre class="ql-syntax" spellcheck="false">options(repr.plot.width = 7, repr.plot.height = 7, repr.plot.res = 250)
</pre><p>You can see an example in the chunk before the bar plots here: <a href="https://nbviewer.org/github/microbiomedata/nmdc_notebooks/blob/main/taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb" target="_blank">https://nbviewer.org/github/microbiomedata/nmdc_notebooks/blob/main/taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>good options to know about, thanks! fixed I think</p>